### PR TITLE
feat: add playbook to change service status

### PR DESCRIPTION
**Pull request by ender503**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,3 +63,34 @@ After modify the fullnode variables, overwrite the existing setting:
...
+# restart remote-worker
+$ ansible-playbook manage.yml --limit remote-worker -e systemd_status=restarted
+```
+
+To restart service by host:
+
+```bash
+# restart node
+$ ansible-playbook manage.yml --limit node -e systemd_status=restarted
+# restart remote-worker-1
```
Comment (jserv) [inline README.md]: Why suffix `-1`?

Comment (ender503) [reply README.md]: In the `inventory`, we define a `remote-worker` group and that includes 4 remote-worker, `remote-worker-1` is one of them.

```
[remote-worker]
remote-worker-1 ansible_host=192.168.0.101 ansible_user=ubuntu
remote-worker-2 ansible_host=192.168.0.102 ansible_user=ubuntu
remote-worker-3 ansible_host=192.168.0.103 ansible_user=ubuntu
remote-worker-4 ansible_host=192.168.0.104 ansible_user=ubuntu
```

Comment (jserv) [reply README.md]: Since it is a matter of the range of available workers, the implication should be denoted.

Comment (marktwtn) [reply README.md]: I think these can be simplified.

---

For example:

To change all service status:

```bash
$ ansible-playbook manage.yml -e systemd_status=[status]
```
The acceptable **[status]** are
- `started`
- `restarted`
- `stopped`

---

The others can be modified as well.

File: manage.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        name: "rabbitmq"
+        state: '{{ systemd_status }}'
+      when: systemd_status is defined and systemd_status is string
+
+- hosts: fullnode
+  gather_facts: true
+  become: true
+  become_method: sudo
+  tasks:
+    - name: change rabbitmq status
```
Comment (marktwtn) [inline manage.yml]: Please revise to the correct name.

File: manage.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        name: "iri"
+        state: '{{ systemd_status }}'
+      when: systemd_status is defined and systemd_status is string
+
+- hosts: remote-worker
+  gather_facts: true
+  become: true
+  become_method: sudo
+  tasks:
+  - name: change rabbitmq status
```
Comment (marktwtn) [inline manage.yml]: Ditto.

**Review by marktwtn — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -63,3 +63,34 @@ After modify the fullnode variables, overwrite the existing setting:
...
+To change all service status:
+
+```bash
+# restart service
+$ ansible-playbook manage.yml -e systemd_status=restarted
+# start service
+$ ansible-playbook manage.yml -e systemd_status=started
+# stop service
+$ ansible-playbook manage.yml -e systemd_status=stopped
+```
```
Comment (marktwtn) [inline README.md]: I think these can be simplified.

---

For example:

To change all service status:

```bash
$ ansible-playbook manage.yml -e systemd_status=[status]
```
The acceptable **[status]** are
- `started`
- `restarted`
- `stopped`

---

The others can be modified as well.

File: manage.yml (hunk 3 above)
Comment (ender503) [reply manage.yml]: FIxed. 👍

File: manage.yml (hunk 2 above)
Comment (ender503) [reply manage.yml]: FIxed. 👍

File: README.md (hunk 1 above)
Comment (ender503) [reply README.md]: I removed this section, and make it simplified as @marktwtn suggestion.

File: README.md (hunk 4 above)
Comment (ender503) [reply README.md]: Updated. 👍

**Comment by marktwtn** [comment]
The pull request looks good.

**Pull request merged**